src/Data/Textile/Step.elm

```diff
@@ -382,20 +382,22 @@ updateFromInputs { wellKnown } inputs ({ label, country, complementsImpacts } as
 
         Label.Ennobling ->
             { step
-                | dyeingMedium = dyeingMedium
+                | dyeingProcessType = dyeingProcessType
                 , printing = printing
                 , processInfo =
                     { defaultProcessInfo
                         | countryElec = Just country.electricityProcess.name
                         , countryHeat = Just country.heatProcess.name
                         , dyeing =
-                            wellKnown
-                                |> WellKnown.getDyeingProcess
-                                    (dyeingMedium
-                                        |> Maybe.withDefault inputs.product.dyeing.defaultMedium
-                                    )
-                                |> .name
-                                |> Just
+                            -- FIXME
+                            -- wellKnown
+                            --     |> WellKnown.getDyeingProcess
+                            --         (dyeingProcessType
+                            --             |> Maybe.withDefault inputs.product.dyeing.defaultMedium
+                            --         )
+                            --     |> .name
+                            --     |> Just
+                            Nothing
```

Comment:
Maybe we should elaborate a little bit more in the comment about what should be fixed and why/how?

Reply:
It's a leftover, will remove